cloudant-sync-datastore-core/src/main/java/com/cloudant/mazha/CouchConfig.java

```diff
@@ -35,37 +38,23 @@
     // a reverse proxy
     private URI rootUri;
 
-    // Optional custom headers
-    private Map<String, String> customHeaders;
+    private List<HttpConnectionRequestInterceptor> requestInterceptors = new ArrayList
+            <HttpConnectionRequestInterceptor>();
+    private List<HttpConnectionResponseInterceptor> responseInterceptors = new ArrayList
+            <HttpConnectionResponseInterceptor>();
 
-    private List<HttpConnectionRequestInterceptor> requestInterceptors;
-    private List<HttpConnectionResponseInterceptor> responseInterceptors;
+    private HttpConnectionRequestInterceptor headerInterceptor;
```

Comment:
Is this field actually used? Isn't the headerInterceptor just added into the list of `requestInterceptors`?